gui/src/Collaborators/components/projectMembers.tsx

```diff
@@ -43,13 +44,17 @@ export class ProjectMembers extends React.Component<Props & DispatchProps, any>
     this.props.fetchMembersForProject();
   }
 
+  onLoadLabelsClicked = (e: MouseEvent) => {
+    this.props.loadLabelsTab();
+  };
+
   render(): React.ReactElement<HTMLDivElement> {
     return (
       <div className="projectMembers">
         <p></p>
         <h1>Members</h1>
         <p className="lede">Theses are users who have access to this project, meaning they can see data, models and services associated with the project. Additionally, owners and collaborators can create new models, and new services based on those models.</p>
-        <p className="lede">Labels associated with projects have <span className="link" onClick={ this.props.loadLabelsTab }>their own access controls, shown here</span>.</p>
+        <p className="lede">Labels associated with projects have <span className="link" onClick={ this.onLoadLabelsClicked }>their own access controls, shown here</span>.</p>
```

Comment:
Does 2.0+ not allow calling of a function passed from a prop directly? Seems like a weird indirection requirement but if that's the case I guess we'll just have to adhere to it.